src/release.js

```diff
@@ -31,6 +31,7 @@ async function release({
   packageFiles = builder['package-files'].default,
   bumpFiles = builder['bump-files'].default,
   defaultBranch = builder['default-branch'].default,
+  repositoryUrl = builder['repository-url'],
```

Comment:
Since there is no default, you don't need this line. `builder['repository-url']` would be an object.

Reply:
You could `repositoryUrl = builder['repository-url']?.default` if you wanted to in case we ever add one.